### PR TITLE
Make sure the ports are valid when parsing portString

### PR DESCRIPTION
**Pull request by miaoyq**
**What this PR does / why we need it**:
We need ensure the ports are valid when parsing `portString`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/kubectl/service_basic.go [hunk 1]
```diff
@@ -89,14 +89,25 @@ func parsePorts(portString string) (int32, intstr.IntOrString, error) {
 	if err != nil {
 		return 0, intstr.FromInt(0), err
 	}
+
+	if err := validation.IsValidPortNum(port); len(err) != 0 {
```
Comment (sttts) [inline pkg/kubectl/service_basic.go]: errs

File: pkg/kubectl/service_basic.go [hunk 2]
```diff
@@ -89,14 +89,25 @@ func parsePorts(portString string) (int32, intstr.IntOrString, error) {
...
+		return 0, intstr.FromInt(0), fmt.Errorf(strings.Join(err, ","))
+	}
+
 	if len(portStringSlice) == 1 {
 		return int32(port), intstr.FromInt(int(port)), nil
 	}
 
 	var targetPort intstr.IntOrString
 	if portNum, err := strconv.Atoi(portStringSlice[1]); err != nil {
+		if err := validation.IsValidPortName(portStringSlice[1]); len(err) != 0 {
```
Comment (sttts) [inline pkg/kubectl/service_basic.go]: errs

File: pkg/kubectl/service_basic.go (hunk 1 above)
Comment (miaoyq) [reply pkg/kubectl/service_basic.go]: Done.

File: pkg/kubectl/service_basic.go (hunk 2 above)
Comment (miaoyq) [reply pkg/kubectl/service_basic.go]: Done.

**Comment by miaoyq** [comment]
@sttts Thank for your review, have addressed.  :-)

**Pull request merged**